docs/source/contribution-guidelines.rst

```diff
@@ -5,3 +5,19 @@ Contributing Guidelines
 "Autometa is an open-source project developed on
 GitHub. If you would like to help develop Autometa or
 have ideas for new features please see our `contributing guidelines <https://github.com/KwanLab/Autometa/blob/master/.github/CONTRIBUTING.md>`__
```

Comment:
Note: We have two different versions of `CONTRIBUTING.md`. The `dev` branch refers to using `black` formatting whereas the master branch does not